cmd/region/instance_types.go

```diff
@@ -63,13 +72,15 @@ var getInstanceTypesCmd = &cobra.Command{
 }
 
 func init() {
-	ClusterCmd.AddCommand(getInstanceTypesCmd)
+	CloudRegionsCmd.AddCommand(InstanceCmd)
+	InstanceCmd.AddCommand(getInstanceTypesCmd)
+
 	getInstanceTypesCmd.Flags().SortFlags = false
 	getInstanceTypesCmd.Flags().String("cloud-provider", "", "[REQUIRED] The cloud provider for which the regions have to be fetched. AWS or GCP.")
 	getInstanceTypesCmd.MarkFlagRequired("cloud-provider")
 	getInstanceTypesCmd.Flags().String("region", "", "[REQUIRED] The region in the cloud provider for which the instance types have to fetched.")
 	getInstanceTypesCmd.MarkFlagRequired("region")
 	getInstanceTypesCmd.Flags().String("tier", "Dedicated", "[OPTIONAL] Tier. Sandbox or Dedicated.")
-	getInstanceTypesCmd.Flags().Bool("show-disabled", false, "[OPTIONAL] Whether to show disabled instance types. true or false.")
+	getInstanceTypesCmd.Flags().Bool("show-disabled", true, "[OPTIONAL] Whether to show disabled instance types. true or false.")
 
```

Comment:
nit: why do we want to show the disabled instance types?

Reply:
Because the UI shows them as well.